### PR TITLE
Enable retry and delay settings

### PR DESCRIPTION
**Pull request by kevinhyang**
@rickhutcheson @udemy/platform 

Part 1 of: EP-981

### CODE REVIEW

File: django_dbconn_retry/__init__.py [hunk 1]
```diff
@@ -53,15 +54,21 @@ def ensure_connection_with_retries(self: django_db_base.BaseDatabaseWrapper) ->
                     self.connect()
                 except Exception as e:
                     if isinstance(e, _operror_types):
-                        if hasattr(self, "_connection_retries") and self._connection_retries >= 1:
+                        if not hasattr(self, "_connection_retries"):
+                            self._connection_retries = 0
+                        if self._connection_retries >= self.settings_dict.get("MAX_RETRIES", 1):
```
Comment (rickhutcheson) [inline django_dbconn_retry/__init__.py]: let's default to 0

File: django_dbconn_retry/__init__.py [hunk 2]
```diff
@@ -53,15 +54,21 @@ def ensure_connection_with_retries(self: django_db_base.BaseDatabaseWrapper) ->
...
                             _log.error("Reconnecting to the database didn't help %s", str(e))
                             del self._in_connecting
                             post_reconnect.send(self.__class__, dbwrapper=self)
                             raise
                         else:
+                            if self._connection_retries > 0:
+                                retry_delay = self.settings_dict.get("SUBSEQUENT_RETRY_DELAY", None)
```
Comment (rickhutcheson) [inline django_dbconn_retry/__init__.py]: CONNECTION_RETRY_DELAY?

File: django_dbconn_retry/__init__.py (hunk 1 above)
Comment (rickhutcheson) [reply django_dbconn_retry/__init__.py]: CONNECTION_RETRY_MAX_ATTTEMPTS?

**Review by rickhutcheson — APPROVED** [review]
Couple suggestions about using a common prefix (CONNECTION_RETRY) so the settings are easy to identify. Otherwise lgtm :heavy_check_mark:

**Comment by kevinhyang** [comment]
Decided to go with `DBCONN_RETRY_` since that's what this library is called 😛

**Comment by rickhutcheson** [comment]
I like that too =]

**Pull request merged**